qtdialogs/WalletMigrationSignals.py

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+#                                                                            #
+#  Copyright (C) 2025, goatpig                                               #
+#  Distributed under the MIT license                                         #
+#  See LICENSE-MIT or https://opensource.org/licenses/MIT                    #
+#                                                                            #
+##############################################################################
+
+from qtpy import QtCore, QtWidgets
```

Comment:
I dont understand the purpose of this file:
1. the signals are not used
2. there are unused imports
3. the passphrase reuse signal is silly, it can simply be checked when clicking next
4. the connectSignals method can implemented as is in DlgWalletMigration, why have an entire class for it, let alone a file?